pkg/apis/core/validation/validation.go

```diff
@@ -6695,8 +6814,14 @@ func validateBasicResource(quantity resource.Quantity, fldPath *field.Path) fiel
 	return field.ErrorList{}
 }
 
+// ResourceValidationOptions
+type ResourceValidationOptions struct {
+	PodLevelResource                bool
+	AllowIndivisibleHugePagesValues bool
```

Comment:
Worth documenting the opt added VS PodValidationOptions.AllowIndivisibleHugePagesValues. Looks like they always share the same value?

Reply:
removed the ResourceValidationOptions struct and refactored the code.